src/main/java/com/training/springpractice/controllers/CatalogController.java

```diff
@@ -0,0 +1,64 @@
+package com.training.springpractice.controllers;
+
+import com.training.springpractice.models.Catalog;
+import com.training.springpractice.models.CatalogBody;
+import com.training.springpractice.services.CatalogService;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.PropertyMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
```

Comment:
There is another anotation **@Controlle**r, what is the difference between **Controller** and **RestController** and why you decided to use the 2nd one?